python/sglang/srt/mm_utils.py

```diff
@@ -3,7 +3,7 @@
 import base64
 import math
 from io import BytesIO
-
+import re
```

Comment:
fix the linting check by https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md#format-your-code